email/1.3/modules/ROOT/edits/email-trigger.adoc

```diff
@@ -0,0 +1,219 @@
+= Trigger a Flow When Receiving a New Email Examples - Mule 4
+:page-aliases: connectors::email/email-trigger.adoc
+
+The following examples illustrate how you configure the Anypoint Connector for Email (Email Connector) source listeners *On New Email - IMAP* and *On New Email - POP3* to identify and retrieve unread email messages, initiating a flow.
+
+== Configure Delete After Retrieve Parameter
+
+Email Connector *On New Email - IMAP* and *On New Email - POP3* sources support the *Delete after retrieve* parameter that enables the deletion of retrieved emails from the mailbox folder after the operation process is complete. By deleting processed emails, you can ensure that the next processed emails are new ones. Because the POP3 protocol doesn't support watermarking, the *Delete after retrieve* parameter is commonly used with this protocol. +
+The parameter is disabled by default to avoid deleting emails by mistake.
+
+The following example shows how to configure this parameter in Anypoint Studio:
+
+. Select the *On New Email - IMAP* or *On New Email - POP3* source in the Studio canvas.
+. In the source configuration screen, enable the *Delete after retrieve* parameter.
+
+The following screenshot shows the *Delete after retrieve* configuration for the *On New Email - IMAP* source:
+
+.Delete after retrieve with On New Email - IMAP operation
+image::email-delete-1.png[Delete after retrieve with On New Email - IMAP]
+
+In the XML editor, the configuration `deleteAfterRetrieve="true"` is under the `listener-imap` tag:
+[source, xml, linenums]
+----
+<flow name="OnNewEmail-IMAP">
+		<email:listener-imap config-ref="Email_IMAP" watermarkEnabled="false" deleteAfterRetrieve="true">
+			<scheduling-strategy >
+				<fixed-frequency />
+			</scheduling-strategy>
+		</email:listener-imap>
+	</flow>
+----
+
+The following screenshot shows the *Delete after retrieve* configuration for the *On New Email - POP3* source:
+
+.Delete after retrieve with On New Email - POP3 operation
+image::email-delete-2.png[Delete after retrieve with On New Email - POP]
+
+In the XML editor, the configuration `deleteAfterRetrieve="true"` is under the `listener-pop3` tag:
+[source, xml, linenums]
+----
+<flow name="OnNewEmail-POP3">
+  <email:listener-pop3 config-ref="Email_POP3" deleteAfterRetrieve="true">
+    <scheduling-strategy >
+      <fixed-frequency />
+    </scheduling-strategy>
+  </email:listener-pop3>
+</flow>
+----
+
+
+== Configure the Enable Watermark Parameter
```

Comment:
Ah! Now that I see this, I wonder if the intro text that follows the L1 title could include something about the need to identify separate operations in a stream of incoming email, and that there are two methods: deleting email as it is used, and watermarking.
We could even state in the previous subtopic's beginning that although IMAP supports both methods, POP3 supports only Delete.

Reply:
Dang it! I'm sorry...here I go performing CU again.
Okay, I swear...I am going to focus on CE and not format or wording options from this point onward!